lib/bootstrap_form/form_builder.rb

```diff
@@ -23,241 +46,16 @@ def initialize(object_name, object, template, options)
       @label_col = options[:label_col] || default_label_col
       @control_col = options[:control_col] || default_control_col
       @label_errors = options[:label_errors] || false
+
       @inline_errors = if options[:inline_errors].nil?
                          @label_errors != true
                        else
                          options[:inline_errors] != false
                        end
       @acts_like_form_tag = options[:acts_like_form_tag]
-
       super
     end
 
-    FIELD_HELPERS.each do |method_name|
```

Comment:
I'm curious why you chose to remove the enumeration here and on line 51? I like what you did in that it's consistent, but I also like the fact that I can quickly tell from the code which methods behave the same with respect to their options.

Reply:
We could have left it in, and I did consider it. My decision stemmed from being able to clearly see what field inputs there are. To the user, it doesn't matter what the implementation is (mostly). Just that it works, and by seeing what field types there are clearly, it helps with understanding how to use the gem.

Reply:
That makes sense. I totally agree that the implementation shouldn't matter to the user (programmer using bootstrap_form). I like what you've done.